core/src/avm2/globals/flash/ui/ContextMenuBuiltInItems.as

```diff
@@ -1,13 +1,97 @@
 package flash.ui
 {
-	public final class ContextMenuBuiltInItems {
-		public var forwardAndBack:Boolean = true;
-		public var loop:Boolean = true;
-		public var play:Boolean = true;
-		public var print:Boolean = true;
-		public var quality:Boolean = true;
-		public var rewind:Boolean = true;
-		public var save:Boolean = true;
-		public var zoom:Boolean = true;
-	}
+    public final class ContextMenuBuiltInItems {
+        private var _forwardAndBack:Boolean = true;
```

Comment:
a bit late but: was there a reason to convert these?
I'm tempted to convert back ;D

Reply:
These are technically correct to flash (see the above review by Dinnerbone). I don't know why Flash does it this way.